indigo_eln_script.yaml

```diff
@@ -0,0 +1,935 @@
+    Type: String
+    Default: public.ecr.aws/m5k0g6n7/indigo_eln:signature-20250121-214015
+  IndigoCrsServiceTag:
+    Description: Indigo Crs Service Docker image tag
+    Type: String
+    Default: public.ecr.aws/m5k0g6n7/indigo_eln:crs-20250108-040611
+  IndigoMongoServiceTag:
+    Description: Indigo Mongo Service Docker image tag
+    Type: String
+    Default: mongo:5
+  IndigoPostgresServiceTag:
+    Description: Indigo Postgres Service Docker image tag
+    Type: String
+    Default: public.ecr.aws/m5k0g6n7/indigo_eln:postgres-20250121-214015
+  IndigoPostgresCrsServiceTag:
+    Description: Indigo Postgres Crs Service Docker image tag
+    Type: String
+    Default: public.ecr.aws/m5k0g6n7/indigo_eln:postgres-crs-20250108-040611
+  PermissionsBoundaryArn:
+    Type: String
+    Description: Enter ARN for roles permission boundary
+    Default: arn:aws:iam::12345678:policy/some_role_boundary # UPDATEME
+  KeyPairName:
+    Description: >-
+      Name of an existing Amazon EC2 key pair to enable SSH or RDP access to the
+      instances.
+    Type: String
+    MinLength: '1'
+    MaxLength: '255'
+    AllowedPattern: '[ -~]*'
+    ConstraintDescription: KeyPairName is a required Field and can contain only ASCII characters.
+    Default: indigo_service # UPDATEME
+  VPCId:
+    Type: String
+    Description: VPC to use
+    Default: vpc-12345678 # UPDATEME
+  HostedZoneId:
+    Description: Route53 Hosted zone id
+    Type: String
+    Default: Z12345678 # UPDATEME
+  IndigoELNDomainName:
+    Type: String
+    Description: Host without http|https protocol.
+    MinLength: '1'
+    ConstraintDescription: IndigoELNDomainName is a required Field
+    Default: indigo-eln.test.lifescience.opensource.epam.com # UPDATEME
+  IndigoELNAPIDomainName:
+    Type: String
+    Description: Host without http|https protocol.
+    MinLength: '1'
+    ConstraintDescription: IndigoELNDomainName is a required Field
+    Default: api.indigo-eln.test.lifescience.opensource.epam.com # UPDATEME
+  IndigoELNContentBucketName:
+    Type: String
+    Description: S3 bucket where frontend code will be served from
+    MinLength: '1'
+    ConstraintDescription: IndigoELNContentBucketName is a required field
+    Default: indigo-eln-artifacts # UPDATEME
+  IndigoELNBuildLogsBucketName:
+    Type: String
+    Description: S3 bucket where build logs are stored
+    MinLength: '1'
+    ConstraintDescription: IndigoELNBuildLogsBucketName is a required field
+    Default: indigo-eln-build-logs # UPDATEME
+  MongoDBPassword:
+    Type: String
+    Description: MongoDB password
+    MinLength: '1'
+    ConstraintDescription: MongoDBPassword is a required field
+    Default: changeit # UPDATEME
+  DefaultAdminPassword:
+    Type: String
+    Description: Default admin password for the application, BCrypt encoded; default is encrypted for "admin"
+    Default: "$2a$10$jawSRFtkzurLE2mDbiAK0O3MDtxvBi1OlbJV8HLFKKpIUC/I7Gyrq" # UPDATEME
```

Comment:
remove the default password, and may be change to password type

Reply:
Added NoEcho to somewhat hide it